Formula/blast.rb

```diff
@@ -0,0 +1,41 @@
+class Blast < Formula
+  desc "Basic Local Alignment Search Tool"
+  homepage "https://blast.ncbi.nlm.nih.gov/"
+  url "https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.6.0/ncbi-blast-2.6.0+-src.tar.gz"
+  mirror "ftp://ftp.hgc.jp/pub/mirror/ncbi/blast/executables/blast+/2.6.0/ncbi-blast-2.6.0+-src.tar.gz"
+  version "2.6.0"
+  sha256 "0510e1d607d0fb4389eca50d434d5a0be787423b6850b3a4f315abc2ef19c996"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  patch do
+    # Fixed upstream in future version > 2.6
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/master/blast/blast-make-fix2.5.0.diff"
```

Comment:
Don't use `master` in the URL.

Reply:
:+1:

Reply:
Replaced with https://raw.githubusercontent.com/Homebrew/formula-patches/5fa7fda60c17a2a31e402522d0f7e5584d3b946b/blast/blast-make-fix2.5.0.diff